scaper/core.py

```diff
@@ -1675,6 +1731,17 @@ def _generate_audio(self, audio_path, ann, reverb=None,
             When True (default), warnings from the pysox module are suppressed
             unless their level is ``'CRITICAL'``.
 
+        Returns
+        -------
+        soundscape_audio : np.ndarray
+            The audio samples of the generated soundscape
+        event_audio_list: list
+            A list of np.ndarrays containing the audio samples of every
+            individual background and foreground sound event. Events are listed
+            in the same order in which they appear in the jams annotations data
+            list, and can be matched with:
+            `for obs, event_audio in zip(ann.data, event_audio_list): ...`.
```

Comment:
Let's make sure this works, and also renders okay in the docs!

Reply:
perhaps as part of my proposed "examples PR" ?